packages/sidenav/src/SidenavItem.ts

```diff
@@ -129,22 +132,32 @@ export class SideNavItem extends LikeAnchor(Focusable) {
                 aria-current=${ifDefined(
                     this.selected && this.href ? 'page' : undefined
                 )}
+                aria-expanded=${ifDefined(
+                    this.hasChildren ? this.expanded : undefined
+                )}
+                aria-controls=${ifDefined(
+                    this.hasChildren && this.expanded ? 'list' : undefined
+                )}
             >
                 <slot name="icon"></slot>
                 ${this.label}
                 <slot></slot>
             </a>
             ${this.expanded
                 ? html`
-                      <slot name="descendant"></slot>
+                      <div id="list" aria-labelledby="item-link" role="list">
+                          <slot name="descendant"></slot>
+                      </div>
                   `
                 : html``}
         `;
     }
 
     protected override updated(changes: PropertyValues): void {
         if (this.hasChildren && this.expanded && !this.selected) {
-            this.focusElement.tabIndex = -1;
+            this.focusElement.tabIndex = 0;
```

Comment:
This allows you to tab into any item, rather the leverage roving tab index. Was that indended?

Reply:
We want to be able to focus the item that toggles expand/collapse of children.

Reply:
Do we want it _in_ the tab order? Or just about to do it programmatically through the Roving Tab Stop pattern? Making sure I understand the appropriate accessibility model here.

Reply:
I think it's more complicated than that. When `manageTabIndex` is `false`, we want to be able to navigate to the parent item when the children are expanded, so that it can be collapsed. But, for the `manageTabIndex` use case, we need to account for when the selected item is a child of a collapsed item, which I address below:

https://github.com/adobe/spectrum-web-components/pull/3352/files#diff-0343271c00070ee3f19ceebd95378dd6c0d7c2a115c3bceb3cb082721ea6f226R56-R77
```ts
    rovingTabindexController = new RovingTabindexController<SideNavItem>(this, {
        focusInIndex: (elements: SideNavItem[]) => {
            let parentSideNavItem: SideNavItem | undefined;
            let index = elements.findIndex((el) => {
                // If the selected item's parent is collapsed, save it for later.
                if (el.value === this.value && this.isDisabledChild(el)) {
                    parentSideNavItem = el.closest(
                        'sp-sidenav-item:not([expanded])'
                    ) as SideNavItem;
                }
                return this.value
                    ? !el.disabled &&
                          !this.isDisabledChild(el) &&
                          el.value === this.value
                    : !el.disabled && !this.isDisabledChild(el);
            });
            // If the selected item's parent is collapsed, focus the collapsed parent.
            if (index === -1 && parentSideNavItem) {
                index = elements.findIndex((el) => el === parentSideNavItem);
            }
            return index;
        },
        direction: 'vertical',
        elements: () =>
            [...this.querySelectorAll('sp-sidenav-item')] as SideNavItem[],
        isFocusableElement: (el: SideNavItem) =>
            !el.disabled && !this.isDisabledChild(el),
    });
```